tests/functional/test1/dag.yaml

```diff
@@ -0,0 +1,18 @@
+task_name: Tester
+slurm_params: "--partition=milano --account=lcls:data --ntasks=1"
+next:
+- task_name: BinaryTester
+  slurm_params: "--partition=milano --account=lcls:data --ntasks=5"
+  next: []
+- task_name: BinaryErrTester
+  slurm_params: "--partition=milano --account=lcls:data --ntasks=5"
+  next: []
+- task_name: SocketTester
+  slurm_params: "--partition=milano --account=lcls:data --ntasks=1"
+  next:
+    - task_name: WriteTester
```

Comment:
These dont need to be defined in the yaml config because they are fully defined here: https://github.com/gadorlhiac/lute_dorlhiac/blob/ff5833da99000789d8661ea988a924333a87e512/lute/tasks/test.py#L78
Is that right?